KernelStubs.php

```diff
@@ -931,6 +931,31 @@ class MediaManager
     {
         private static $medias = [];
 
+        public static function createMedia(int $MediaID): void
+        {
+            self::$medias[$MediaID] = [
```

Comment:
Magst du hier alle in der Doku definierten Felder hinzufügen mit den den Standardwerten? (und MediaType s.o.)